test/Core.Test/OrganizationFeatures/OrganizationSubscriptionUpdate/UpdateSecretsManagerSubscriptionCommandTests.cs

```diff
@@ -17,711 +18,617 @@
 namespace Bit.Core.Test.OrganizationFeatures.OrganizationSubscriptionUpdate;
 
 [SutProviderCustomize]
+[SecretsManagerOrganizationCustomize]
 public class UpdateSecretsManagerSubscriptionCommandTests
 {
+    #region General
```

Comment:
We had a vote and `#regions` lost https://contributing.bitwarden.com/contributing/code-style/#code-regions

Reply:
I forgot about that. I removed the regions, they're probably not necessary, and I'm not sure I want to split the test code into separate files.